README.md

```diff
@@ -274,6 +281,28 @@ specify per-container scope with `systemd_unit_scope` in `podman_kube_specs`. By
 default, rootless containers will use `user` and root containers will use
 `system`.
 
+### podman_activate_systemd_units
+
+Activate each systemd unit as soon as it is created.  The default is `true`.
+You can also do this on a per-unit basis by using `activate_systemd_units` in
+the spec for each unit. For example, if you are deploying several specs, and you
+only want the last one in the list to activate which will trigger the others to
+activate via dependencies, then set `activate_systemd_unit: false` for each one
+except the last one uses `activate_systemd_unit: true`
+
+### podman_pull_image
+
+Ensure that each image mentioned in a kube or quadlet spec is present by pulling
+the image before it is used.  The default is `true`.  Use `false` if the managed
+node already has the correct version, or is not able to pull images.  You can also
+specify this on a per-unit basis with `pull_image`.
```

Comment:
Are there limitations?  Does it work for .kube files?

Reply:
I updated the README

Reply:
LGTM